atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -87,6 +87,10 @@ public Client create() throws Exception {
         }
     }
 
+    private CassandraClient instrumentClient(Client client) {
+        return new CassandraClientImpl(client);
```

Comment:
I assume there will be a follow on to wrap this instrumentation

Reply:
Yep. This is what I'm doing in the next PR.